Sources/Actions/Memberlist.php

```diff
@@ -294,30 +294,7 @@ public function all(): void
 			$_REQUEST['sort'] = 'real_name';
 		}
 
-		if (!is_numeric($_REQUEST['start'])) {
-			if (preg_match('~^[^\'\\\\/]~u', Utils::strtolower($_REQUEST['start']), $match) === 0) {
-				ErrorHandler::fatal('Are you a wannabe hacker?', false);
-			}
-
-			$_REQUEST['start'] = $match[0];
-
-			$request = Db::$db->query(
-				'substring',
-				'SELECT COUNT(*)
-				FROM {db_prefix}members
-				WHERE LOWER(SUBSTRING(real_name, 1, 1)) < {string:first_letter}
-					AND is_activated = {int:is_activated}',
-				[
-					'is_activated' => User::ACTIVATED,
-					'first_letter' => $_REQUEST['start'],
-				],
-			);
-			list($start) = Db::$db->fetch_row($request);
-			$start = (int) $start;
-			Db::$db->free_result($request);
-		} else {
-			$start = (int) $_REQUEST['start'];
-		}
+		$start = (int) $_REQUEST['start'];
 
 		Utils::$context['letter_links'] = '';
```

Comment:
gonna get rid of this, too?

https://github.com/SimpleMachines/SMF/blob/6a949d59fc356e456845e440af952c1840815d27/Sources/Actions/Memberlist.php#L325